### PR TITLE
doc: translating docs/index and docs/ES6 to pt-BR

### PR DESCRIPTION
**Pull request by danielbdias**
Hi everyone!

I'm helping @nodejs/nodejs-pt community to translate some docs to Brazilian Portuguese.

In this PR, I've translated two pages: the index of the docs and the ES6 docs.

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/nodejs-pt

File: locale/pt-br/docs/es6.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Todas as funcionalidades do ECMAScript 2015 (ES6) são divididas em três grupos:
+
+* Todas as funcionalidades em **lançamento** (shipping), que o V8 considera estáveis, são **ativadas por padrão no Node.js** e **NÃO** necessitam de nenhum tipo de flag de tempo de execução.
+* As funcionalidades **em homologação**, que estão quase completas, mas não são consideradas estáveis pelo time do V8 necessitam da flag `--harmony`.
+* As funcionalidades **em progresso** podem ser ativadas individualmente por sua respectiva flag "harmony", entretanto isto é altamente desencorajado a menos que seja para propósito de teste. Nota: estas flags são expostas pelo V8 e podem mudar sem aviso.
+
+## Quais funcionalidades são embarcadas com cada versão do Node.js por padrão?
+
+O site [node.green](http://node.green) tem uma excelente visão sobre quais são as funcionalidades do ECMAScript suportadas em diversas versões do Node.js, baseadas na [tabela de compatibilidade](https://github.com/kangax/compat-table) de kangax.
```
Comment (Drieger) [inline locale/pt-br/docs/es6.md]:
```suggestion
O site [node.green](http://node.green) oferece uma excelente visão sobre quais são as funcionalidades do ECMAScript suportadas em diversas versões do Node.js, baseadas na [tabela de compatibilidade](https://github.com/kangax/compat-table) de kangax.
```

File: locale/pt-br/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  </ul>
+</div>
+
+### Funcionalidades do ES6
+
+A [seção do ES6](/pt-br/docs/es6/) descreve os três grupos de funcionalidade e detalha quais funcionalidades estão habilitadas por padrão no Node.js, incluindo links com explicações. Ela também mostra como encontrar qual versão do V8 está embarcada em conjunto com uma versão particular do Node.js.
+
+### Guias
+
+A [seção de Guias](/pt-br/docs/guides/) têm uma longa lista de artigos densos sobre as funcionalidades técnicas e capacidades do Node.js.features and capabilities.
```
Comment (Drieger) [inline locale/pt-br/docs/index.md]:
```suggestion
A [seção de Guias](/pt-br/docs/guides/) têm uma longa lista de artigos que detalham as funcionalidades técnicas e capacidades do Node.js.
```

**Review by Drieger — CHANGES_REQUESTED** [review]
I left some suggestions, but it is pretty good. The only real request is to remove the English words left in the last paragraph.

Great job @danielbdias.

**Comment by danielbdias** [comment]
Thanks @Drieger ! I've commited your suggestions in the code!

File: locale/pt-br/docs/es6.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+O trabalho de otimizar as funcionalidades introduzidas com o ES2015 e além são coordenadas via um [plano de performance](https://docs.google.com/document/d/1EA9EbfnydAmmU_lM8R_uEMQ-U_v4l9zulePSBkeYWmY), onde o time do V8 encontra e coordena áreas que necessitam de melhorias, além de criar documentos para atacar esses problemas.
+
+## Eu tenho a minha infraestrutura configurada para suportar a flag --harmony. Eu devo remove-la?
+
+O comportamento atual da flag `--harmony` é habilitar somente funcionalidades **em homologação**.  Além disso, ela agora é um sinônimo da flag `--es_staging`. Como mencionado anteriormente, estas funcionalidades estão completas, mas não foram consideradas estáveis ainda. Se você deseja rodar sua aplicação de forma segura, especiamente em ambientes de produção, considere remover esta flag até que ela seja lançada por padrão no V8 e, consequentemente, no Node.js. Se você deseja a manter habilitada, você deve estar preparado para atualizações futuras do Node.js que podem quebrar o seu código caso o V8 mude a sua semântica para seguir o padrão de maneira mais fiel.
```
Comment (TiagoDanin) [inline locale/pt-br/docs/es6.md]: Replace to "Especialmente"

File: locale/pt-br/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: Docs
+layout: docs.hbs
+labels:
+  lts: LTS
+---
+
+# Sobre a Documentação
+
+Existe diversos tipos de documentação disponíveis neste site:
```
Comment (TiagoDanin) [inline locale/pt-br/docs/index.md]: Replace to "Existem"

File: locale/pt-br/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+* Documentação de referência de API
+* Funcionalidades do ES6
+* Guias
+
+### Documentação de referência de API
+
+A [Documentação de referência de API](/api/) provê uma informação detalhada sobre uma função ou objeto no Node.js. Esta documentação indica quais argumentos um método aceita, o valor de retorno deste método e quais erros podem estar relacionados aquele método. Ela também indica quais métodos estão disponíveis para diferentes versões do Node.js.
+
+Esta documentação descreve os módulos embarcadas (built-in) providos pelo Node.js. Ela não documenta os módulos providos pela comunidade.
```
Comment (TiagoDanin) [inline locale/pt-br/docs/index.md]: Replace to "Embarcados"

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

File: locale/pt-br/docs/index.md (hunk 4 above)
Comment (danielbdias) [reply locale/pt-br/docs/index.md]: Typo fixed!

File: locale/pt-br/docs/index.md (hunk 5 above)
Comment (danielbdias) [reply locale/pt-br/docs/index.md]: Typo fixed!

File: locale/pt-br/docs/es6.md (hunk 3 above)
Comment (danielbdias) [reply locale/pt-br/docs/es6.md]: Typo fixed!

**Comment by Drieger** [comment]
LGTM

**Review by Drieger — APPROVED** [review]

**Review by TiagoDanin — APPROVED** [review]
LGTM 💯

**Pull request merged**